Octokit/Models/Request/MilestoneRequest.cs

```diff
@@ -39,8 +39,10 @@ internal string DebuggerDisplay
 
     public enum MilestoneSort
     {
-        [Parameter(Value = "due_date")]
+        [Parameter(Value = "due_on")]
```

Comment:
OK so this is really weird!  The API docs do say `due_on` however I was running some tests and these were now failing.  When I play around with things it looks like actually the docs are incorrect and our codebase was correct - a value of `due_date` provides the correct behavior!
https://api.github.com/repos/ryangribble-test/public-repo-20170506070755114/milestones?sort=due_on&direction=asc
https://api.github.com/repos/ryangribble-test/public-repo-20170506070755114/milestones?sort=due_on&direction=desc
( both of these produce the same response (milestone1, milestone2)

https://api.github.com/repos/ryangribble-test/public-repo-20170506070755114/milestones?sort=due_date&direction=asc
https://api.github.com/repos/ryangribble-test/public-repo-20170506070755114/milestones?sort=due_date&direction=desc

These ones correctly return the expected orders...